scripts/migration-task-script.py

```diff
@@ -7,6 +8,30 @@
 environment = sys.argv[1]
 docker_image = sys.argv[2]
 
+# Check if there are migrations to apply
```

Comment:
Can we maybe move this to a function instead? (To make the script easier to understand)